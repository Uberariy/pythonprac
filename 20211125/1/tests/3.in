@objcount
class A(str):
    pass

c, d, e = A(), A(), A()
print(A.counter)
c = 2100500class A(str):
    pass

c, d, e = A(), A(), A()
print(A.counter)
c = 2100500
print(A.counter)
class A(str):
    pass

c, d, e = A(), A(), A()
print(A.counter)
c = 2100500
print(A.counter)
class A(str):
    pass

c, d, e = A(), A(), A()
print(A.counter)
c = 2100500
print(A.counter)

print(A.counter)
