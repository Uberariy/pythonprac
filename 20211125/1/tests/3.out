@
c = 2
print
print

c, 
c, d
clas
print    pa    pas  pass 21005 = A() A(), A A(), A pass
()
pr(), A((), A((A.cou(A.coun(A.coun)
c = )
prin), A()):
  , A(), , d, e , e = A.count00
pri0500
0class 100500210050:
   = A(),A()
pA(str)ass A(ass A(sc = 210c, d, ecounted, e =er)
clint(A.cnt
clnt(A.cnter)
objcouounterounter)print(Ar)
c =rint(A.s

cs A(strss

str):
t(A.coter)
ter)
tr):
unter)